utils/java/mvn.go

```diff
@@ -123,6 +126,9 @@ func (mdt *MavenDepTreeManager) execMavenDepTree(depTreeExecDir string) (string,
 func (mdt *MavenDepTreeManager) runTreeCmd(depTreeExecDir string) (string, error) {
 	mavenDepTreePath := filepath.Join(depTreeExecDir, mavenDepTreeOutputFile)
 	goals := []string{"com.jfrog:maven-dep-tree:" + mavenDepTreeVersion + ":" + string(Tree), "-DdepsTreeOutputFile=" + mavenDepTreePath, "-B"}
+	if mdt.isCurationCmd {
+		goals = append(goals, "-Dmaven.repo.local="+mdt.curationCacheFolder)
```

Comment:
```suggestion
	if mdt.isCurationCmd && mdt.curationCacheFolder != "" {
		goals = append(goals, "-Dmaven.repo.local="+mdt.curationCacheFolder)
```
what happend if we are setting it to an empty path? can this

Reply:
well its possible to do that from an enviroment variable, thi option by default go to the path under the home folder, If user explicitly set it to be empty then it will go to the default cahce of maven which is under ~/.m2/repository. we set it to be under a different cache folder so the curtion packages downloaded during the build of tree wont infect the default cache(some of them might be blocked by curation policies) so ist kind of not recommended setup.

Reply:
just making sure you are taking into account this case.